corehq/apps/app_manager/suite_xml/case_tile_templates/tdh.txt

```diff
@@ -58,12 +58,17 @@
       </header>
       <template>
         <text>
-          &lt;b&gt;{header[prefix]} &lt;/b&gt; <xpath function="{header[xpath_function]}"/>
+          &lt;b&gt;{header[prefix]} &lt;/b&gt;
+          <xpath function="{header[xpath_function]}">
+            {header[variables]}
```

Comment:
@dannyroberts definitely a real change. There was no option to add xpath variables till now for these fields. So added that ability for all fields and also there sort options.